pyzx/gadget_extract.py

```diff
@@ -7,7 +7,7 @@
 from .simplify import id_simp, tcount
 from .rules import apply_rule, pivot, match_spider_parallel, spider
 from .circuit import Circuit
-from .circuit.gates import Gate, ParityPhase, CNOT, HAD, ZPhase, XPhase, CZ, CX, SWAP, InitAncilla
+from .circuit.gates import Gate, ParityPhase, CNOT, HAD, ZPhase, XPhase, CZ, XCX, SWAP, InitAncilla
```

Comment:
(The `XCX` gate is not actually used in this file, nor are many of the other imported gates, but as the comment at the top says it is experimental, presumably it is intended to be used at some point.)